src/Workspaces/Core/MSBuild/MSBuild/MSBuildWorkspace.cs

```diff
@@ -158,11 +149,15 @@ public void CloseSolution()
             }
         }
 
+#if SUPPORT_TRYAPPLYCHANGES
```

Comment:
wha'ts this?

Reply:
Removed in a later commit -- this was just me temporarily removing this code in one commit while the other parts of the system got working.